sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -228,7 +231,7 @@ var options = new ServiceBusSessionProcessorOptions
 ServiceBusProcessor processor = client.CreateSessionProcessor(queueName);
 ```
 
-You have similar options when working with the receivers. Please note that creating a session receiver is an async operation because the library will need to get a lock on the session by connecting to the service first.
+The below code snippet shows you the session variation of the "receiver". Please note that creating a session receiver is an async operation because the library will need to get a lock on the session by connecting to the service first.
```

Comment:
why quotes?

Reply:
Wanted "receiver" and "processor" to get emphasized as different components than be termed as just another English word. Italics is better?

Reply:
I think you can use the full type name and do code formatting then.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-net/pull/14166/commits/2b526649efee8d5c8dd2331dee0397f4b2598a98